src/bc_common.d

```diff
@@ -102,12 +102,18 @@ enum BCTypeEnum : ubyte
 
 }
 
+enum BCTypeFlags
+{
+  Const = 0x1,
+}
 struct BCType
 {
     BCTypeEnum type;
     alias type this;
     /// 0 means basic type
     uint typeIndex;
+    // additional information
+    uint flags;
```

Comment:
I have no idea, I think the const assignment should be caught before ever getting to the interpreter.